iwyu.cc

```diff
@@ -3933,6 +3933,16 @@ class IwyuAstConsumer
     return Base::VisitTypedefType(type);
   }
 
+  bool VisitEnumDecl(clang::EnumDecl* decl) {
```

Comment:
This wants to move closer to the other `Visit` methods for `Decl`s (there's a comment somewhere saying `// Visitors of types derived from clang::Decl.`).

Also, I think this should have the `CanIgnoreCurrentASTNode()` guard clause like the others.

Reply:
(but as a two-liner, see other PR)